### DISCUSSION TITLE
mKCP status

### DISCUSSION THREAD

**Discussion by zonescape** [XTLS/Xray-core#3306] [Q&A] [answered]
I found [this comment](https://github.com/XTLS/Xray-core/issues/2331#issuecomment-1636724849) by @RPRX (I don't understand Chinese, so the translation may not be correct):

> I plan to add QUIC support to REALITY after the official implementation of QUIC in Golang is released, and then delete the existing QUIC and mKCP transport layers.
> 
> 我打算在 Golang 官方实现的 QUIC 发布后给 REALITY 加上 QUIC 支持，到时还会删掉现有的 QUIC 和 mKCP 传输层

Does this mean that the mKCP transport is obsolete and should not be used?

**Comment by chise0713** [comment, marked as answer]
If there is no problem in using it, please use it as is. Using mKCP is not recommended because it requires more bandwidth to send the same size of data.
